.gitignore

```diff
@@ -9,3 +9,4 @@ Pipfile.lock
 *.kdbx
 *.kdbx.out
 .idea
+.venv
```

Comment:
Not usnig an isolated Python environment rapidly leads to a mess. I've marked the demi-standard `.venv` directory as ignored to make it easier to keep a virtual environment inside the project folder.